tgui/packages/tgui/interfaces/TramController.tsx

```diff
@@ -0,0 +1,279 @@
+import { useBackend, useLocalState } from '../backend';
+import { BooleanLike } from 'common/react';
+import { Section, NoticeBox, Dropdown, LabeledList, ProgressBar, Flex, Button } from '../components';
+import { toFixed } from 'common/math';
+import { Window } from '../layouts';
+
+type TransportData = {
```

Comment:
While I won't knock this behavior I think it's not necessary for a local type to name it so explicitly- just "data" is fine if it's the backend data